packages/opentelemetry-cloud-trace-exporter/src/trace.ts

```diff
@@ -70,22 +68,14 @@ export class TraceExporter implements SpanExporter {
     }
 
     this._logger.debug('Google Cloud Trace export');
-    const authorizedSpans = await this._authorize(
-      spans.map(getReadableSpanTransformer(this._projectId))
-    );
 
-    if (!authorizedSpans) {
-      return resultCallback(ExportResult.FAILED_NOT_RETRYABLE);
-    }
-    this._logger.debug('Google Cloud Trace got span authorization');
+    const namedSpans: NamedSpans = {
+      name: `projects/${this._projectId}`,
+      spans: spans.map(getReadableSpanTransformer(this._projectId)),
+    };
 
-    try {
-      await this._batchWriteSpans(authorizedSpans);
-      resultCallback(ExportResult.SUCCESS);
-    } catch (err) {
-      this._logger.error(`Google Cloud Trace failed to export ${err}`);
-      resultCallback(ExportResult.FAILED_RETRYABLE);
-    }
+    const result = await this._batchWriteSpans(namedSpans);
+    resultCallback(result);
```

Comment:
I dont think its necessary to wrap this in a try-catch because `_batchWriteSpans` now always resolves with an `ExportResult`